offer-manager-domain/src/main/groovy/life/qbic/business/offers/policies/taxes/TaxPolicy.java

```diff
@@ -0,0 +1,28 @@
+package life.qbic.business.offers.policies.taxes;
+
+import java.math.BigDecimal;
+import life.qbic.business.persons.affiliation.AffiliationCategory;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
```

Comment:
```suggestion
 * <b>Tax Policy</b>
 *
 * <p>Abstract class to be extended for tax policies. A tax policy is defined based on an affiliation category and country. It will define what VAT ratio is applied to the offer.</p>
 *
 * @since 1.3.0
```

Reply:
outdated, is an interface now